readme.md

```diff
@@ -1,202 +1,55 @@
 # Objects and Functions
 
-## Learning Objectives (5 / 5)
+[![Build Status](https://travis-ci.org/ga-wdi-lessons/js-objects-functions.svg?branch=master)](https://travis-ci.org/ga-wdi-lessons/js-objects-functions)
```

Comment:
The link references the "master" branch, rather than the branch for this PR.  I don't have an answer for managing that easily.